tests/framework/db/QueryTest.php

```diff
@@ -340,6 +340,9 @@ public function testCount()
         // testing that orderBy() should be ignored here as it does not affect the count anyway.
         $count = (new Query)->from('customer')->orderBy('status')->count('*', $db);
         $this->assertEquals(3, $count);
+        
+        $count = (new Query)->from('customer')->orderBy('id')->limit(1)->count('*', $db);
```

Comment:
`LIMIT` gets ignored on `COUNT` for MySQL so it was to test if a limit was set it still returned 3.
Not sure if this behaviour is different on MSSQL.